examples/tests/end-comment.earth

```diff
@@ -0,0 +1,5 @@
+FROM busybox:1.32.0
+
+test:
+    RUN echo 'Hello, world!'
+    # problematic comment
```

Comment:
Something with both end problem tokens in one line. It makes current earth angery